spark/src/test/scala/org/apache/spark/sql/delta/test/DeltaSQLTestUtils.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
```

Comment:
DYK if we're supposed to bump the year on new files or if we expect 2021 every time? cc: @scottsand-db ?